### ISSUE TITLE
Add support for SQL lower

### ISSUE THREAD

**Issue by charlesmarvin** [vincentlauvlwj/Ktorm#109]
Any plans to support the `lower(<COLUMN>)` function? I imagine it can be implemented as follows:

```
fun lower(col: ColumnDeclaring<String>) = FunctionExpression(
    functionName = "lower",
    arguments = listOf(col.asExpression()),
    sqlType = VarcharSqlType
)
```

**Comment by vincentlauvlwj** [comment]
Added, will release in next version.

**Comment by vincentlauvlwj** [comment]
Released in version 3.0.0

**Comment by xiaoyv404** [comment]
postgres also support lower
Any plans to support the function for postgres?
